openfga/v1/openfga_service.proto

```diff
@@ -774,6 +774,18 @@ service OpenFGAService {
   }
 }
 
+// Controls the consistency preferences when calling the query APIs.
+// buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX
+// buf:lint:ignore ENUM_VALUE_PREFIX
+enum ConsistencyPreference {
+  // Default if not set. Behavior will be the same as MINIMIZE_LATENCY
+  UNSPECIFIED = 0;
+  // Minimize latency at the potential expense of higher consistency.
+  MINIMIZE_LATENCY = 100;
```

Comment:
Why starting on `100` instead of `1`?

Reply:
We [explicitly chose](https://github.com/openfga/api/pull/170#discussion_r1628702672) to not use `1`/`2` to allow us the choice to expand these options in future easily. I believe the intent is that if we have an option that is between the behaviour of `MINIMIZE_LATENCY` and `HIGHER_CONSISTENCY` we could choose a number in the range of `101-199` to represent it and it would have more meaning than `3`

Reply:
I also find this to be a little odd. 

> I believe the intent is that if we have an option that is between the behaviour

Conceptually understand but what's "between" these three settings? Like will we have a `REALLY_REALLY_MINIMIZE_LATENCY` value? Obviously can't look into the future but I'd rather take a minute to ponder how this feature could evolve so we can better account for it today.

Reply:
I think `MINIMIZE_LATENCY ` will always be our lowest possible value, beyond that maybe it's feasible that we'd want to skip in-memory cache but not a DB cache which would sit between `MINIMIZE_LATENCY` and `HIGHER_CONSISTENCY`?

@rhamzeh maybe you might have some stronger (or should it be "higher" 😆 ) reasoning from when the decision was made?